openeogeotrellis/load_stac.py

```diff
@@ -84,9 +84,28 @@ def supports_item_search(coll: pystac.Collection) -> bool:
         conforms_to = coll.get_root().extra_fields.get("conformsTo", [])
         return any(conformance_class.endswith("/item-search") for conformance_class in conforms_to)
 
+    def is_raster_mime_type(mime_type: str) -> bool:
+        mime_type = mime_type.lower()
+        # https://github.com/radiantearth/stac-spec/blob/master/best-practices.md#common-media-types-in-stac
+        if (
+            mime_type.startswith("image/tif")  # match both tif and tiff, just in case
+            or mime_type.startswith("image/vnd.stac.geotiff")
+            or mime_type.startswith("image/jp2")
+            or mime_type.startswith("image/png")
+            or mime_type.startswith("image/jpeg")
+            or mime_type.startswith("application/x-hdf")  # matches hdf5 and hdf
+            or mime_type.startswith("application/x-netcdf")
+            or mime_type.startswith("application/netcdf")
+        ):
+            return True
```

Comment:
```suggestion
        return (
            mime_type.startswith("image/tiff")
            or mime_type.startswith("image/vnd.stac.geotiff")
            or mime_type.startswith("image/jp2")
            or mime_type.startswith("image/png")
            or mime_type.startswith("image/jpeg")
            or mime_type.startswith("application/x-hdf")  # matches hdf5 and hdf
            or mime_type.startswith("application/x-netcdf")
            or mime_type.startswith("application/netcdf")
        )
```

Always return a `bool`; `image/tif` does not exist so I would just reject it.

Reply:
thanks, fixed!